### PR TITLE
modify some wrong func information in device_tool.go

### PR DESCRIPTION
**Pull request by lixiaobing1**
modify some wrong func information in device_tool.go.

### CODE REVIEW

File: contrib/docker-device-tool/device_tool.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func main() {
 		}
 		status, err := devices.GetDeviceStatus(args[1])
 		if err != nil {
-			fmt.Println("Can't get device info: ", err)
+			fmt.Println("Can't get device status: ", err)
```
Comment (AkihiroSuda) [inline contrib/docker-device-tool/device_tool.go]: I wonder this fix is too nitpicky, the other fix in L164 LGTM

**Comment by lixiaobing10051267** [comment]
yeah, has been restored the first.

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**